pkg/cmd/completion.go

```diff
@@ -73,7 +73,7 @@ var completionInstallCmd = &cobra.Command{
 		var buf bytes.Buffer
 		var path string
 		var name string
-
+		
```

Comment:
Erroneous addition of whitespace. You'll want to configure your IDE to strip extra whitespace on save.